back-end/developerverse/models.py

```diff
@@ -1,3 +1,29 @@
+from django.conf import settings
 from django.db import models
 
 # Create your models here.
+
+class Project(models.Model):
+
+    # We cannot directly use 
+    user = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        on_delete=models.CASCADE,
+    )

```

Comment:
This comment is a bit confusing. Cannot use what with what?

Reply:
I revised the comment to be more specific and point to relevant documentation from Django